docker/dockerfiles/backend.Dockerfile

```diff
@@ -56,4 +56,4 @@ RUN set -e; \
 
 EXPOSE 8000
 
-ENTRYPOINT [ "./entrypoint.sh" ]
+CMD [ "./entrypoint.sh" ]
```

Comment:
@chandrasekharan-zipstack This should be `ENTRYPOINT` itself so that we can override the same via `CMD` in docker compose files, etc.